splitio/storage/inmemmory.py

```diff
@@ -419,3 +452,263 @@ def clear(self):
         """
         with self._lock:
             self._events = queue.Queue(maxsize=self._queue_size)
+
+class InMemoryTelemetryStorage(TelemetryStorage):
+    """In-memory telemetry storage."""
+
+    def __init__(self):
+        """Constructor"""
+        self._counters = {'iQ': 0, 'iDe': 0, 'iDr': 0, 'eQ': 0, 'eD': 0, 'sL': 0,
+                        'aR': 0, 'tR': 0}
+        self._latencies = {'mL': {'t': [], 'ts': [], 'tc': [], 'tcs': [], 'tr': []},
+                           'hL': {'sp': [], 'se': [], 'ms': [], 'im': [], 'ic': [], 'ev': [], 'te': [], 'to': []}}
+        self._exceptions = {'mE': {'t': 0, 'ts': 0, 'tc': 0, 'tcs': 0, 'tr': 0}}
+        self._records = {'IS': {'sp': 0, 'se': 0, 'ms': 0, 'im': 0, 'ic': 0, 'ev': 0, 'te': 0, 'to': 0},
+                         'sL': 0}
+        self._http_errors = {'sp': {}, 'se': {}, 'ms': {}, 'im': {}, 'ic': {}, 'ev': {}, 'te': {}, 'to': {}}
+        self._streaming_events = []
+        self._tags = []
+        self._integrations = {}
+        self._config = {'bT':0, 'nR':0, 'uC': 0}
+        self._map_latencies = {'Treatment': 't', 'Treatments': 'ts', 'TreatmentWithConfig': 'tc', 'TreatmentsWithConfig': 'tcs', 'Track': 'tr'}
```

Comment:
if we're going to use dicts like this, we should probably have some methods like:

`_reset_counters()`, `_reset_latencies()`, etc.

so that we can call them upon construction and everytime the data gets popped. having the code repetaed it's very easy to miss a key if we do an update of htis logic

Reply:
will do